test/org/sagebionetworks/bridge/models/ClientInfoTest.java

```diff
@@ -226,4 +227,17 @@ public void deprecatedOperatingSystemParsed() {
         ClientInfo info = ClientInfo.parseUserAgentString(VALID_LONG_UA_3_DEPRECATED);
         assertEquals(ANDROID, info.getOsName());
     }
+    
+    @Test
+    public void synonymOsNameConvertedToCanonicalValue() {
+        ClientInfo info = ClientInfo.parseUserAgentString(VALID_LONG_WITH_SYNONYM);
+        
+        assertEquals("Cardio Health", info.getAppName());
+        assertEquals(1, info.getAppVersion().intValue());
+        assertEquals("Unknown iPhone", info.getDeviceName());
+        assertEquals(IOS, info.getOsName());
```

Comment:
you test it with 'iOS' but assert it against 'IOS'. Will parseUserAgentString() automatically convert to upper case?